scripts/touchpoint_api.py

```diff
@@ -0,0 +1,106 @@
+from argparse import ArgumentParser
+import requests
+import json
+import re
+
+# This script calls the touchpoint API to retrieve survey results for the provided document (form_id)
+# You will need to use your own Touchpoint API key:
+# touchpoint.py <form_id> <api_key> <start_date> <end_date>
+# if you want to save to the default file (touchpoint.csv), use the -p flag.
+
+
+
+MAX_PAGES = 500 # this can be changed to throttle if needed
+CSV_HEADER = 'id, date, referer, user agent, page, raw answer, language, english answer\n'
+
+def parse_args():
+    parser = ArgumentParser(
+        fromfile_prefix_chars="@", prog='Touchpoint Parser',
+        description='Parse specific results from a Touchpoint form',
+        epilog='Now, be off and about your business!', conflict_handler='error', add_help=True
+    )
+    parser.add_argument("form_id")
+    parser.add_argument("apikey")
+    parser.add_argument("start_date")
+    parser.add_argument("end_date")
+    parser.add_argument("filename")
```

Comment:
personally I would not do positional arguments, I'd make all of these flags, so they're easier to remember

Reply:
I'm thinking about just making it run off a config file instead, thereby needing none of this nonsense. :)